host/OEHOSTMR/CMakeLists.txt

```diff
@@ -0,0 +1,189 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# Add a library oehostmr independent of libsgx_enclave_common/libsgx_dcap_ql
+if (UNIX)
+  set(PLATFORM_HOST_MR_SRC
+    crypto/openssl/init.c
+    crypto/openssl/key.c
+    crypto/openssl/rsa.c
+    crypto/openssl/sha.c
+    linux/hostthread.c
+    linux/windows.c)
+elseif (WIN32)
+  set(PLATFORM_HOST_MR_SRC
+    crypto/bcrypt/key.c
+    crypto/bcrypt/rsa.c
+    crypto/bcrypt/sha.c
+    crypto/bcrypt/pem.c
+    crypto/bcrypt/util.c
+    windows/hostthread.c
+
+    ../3rdparty/mbedtls/mbedtls/library/bignum.c
+    ../3rdparty/mbedtls/mbedtls/library/platform_util.c #Used by bignum.c
+    ../common/asn1.c
+    ../common/cert.c
+    crypto/bcrypt/cert.c
+    crypto/bcrypt/crl.c
+    crypto/bcrypt/ec.c
+    crypto/bcrypt/hmac.c
+    crypto/bcrypt/key.c
+    crypto/bcrypt/pem.c
+    crypto/bcrypt/random.c
+    crypto/bcrypt/rsa.c
+    crypto/bcrypt/sha.c
+    windows/hostthread.c
+    windows/syscall.c
+    windows/time.c)
+else()
+  message(FATAL_ERROR "Unknown OS. Only supported OSes are Linux and Windows")
+endif()
+
+if (OE_SGX)
+  list(APPEND PLATFORM_HOST_MR_SRC
+    sgx/create.c
+    sgx/elf.c
+    sgx/load.c
+    sgx/loadelf.c
+    sgx/loadpe.c
+    sgx/sgxload.c
+    sgx/sgxmeasure.c
+    sgx/sgxsign.c
+    sgx/sgxtypes.c)
+
+  # OS specific as well.
+  if (UNIX)
+    list(APPEND PLATFORM_HOST_MR_SRC
+      sgx/linux/xstate.c)
+  else()
+    list(APPEND PLATFORM_HOST_MR_SRC
+      sgx/windows/xstate.c)
+  endif()
+
+  set(PLATFORM_FLAGS "-m64")
+elseif(OE_TRUSTZONE)
```

Comment:
nit: use consistent spacing.  Usually this file has a space before (like in line 13 etc) but here doesn't.  Similarly line 97 and 110 have different styles. Pick a style and be consistent throughout this file.

Reply:
Fixed, thanks.